charmhelpers/contrib/openstack/templates/haproxy.cfg

```diff
@@ -82,7 +82,11 @@ backend {{ service }}_{{ frontend }}
     {% endif -%}
     {% endif -%}
     {% for unit, address in frontends[frontend]['backends'].items() -%}
+    {% if https -%}
+    server {{ unit }} {{ address }}:{{ ports[1] }} check check-ssl verify none
```

Comment:
This is essentially adding a tls check on the existing tcp check. Would it be better to have an actual http check here?

Reply:
ah ignore me, I see that this is done in the radosgw charm via the backend options. Apologies for the noise